.github/workflows/examples.yml

```diff
@@ -186,7 +187,7 @@ jobs:
         wasmedge --dir .:. target/wasm32-wasi/release/examples/js_extend.wasm
 
     - name: Node fs module test
-      #timeout-minutes: 10
+      timeout-minutes: 60
```

Comment:
Why do we need to increase this to 60?

Reply:
This test consistently takes a long time, this time it's 11 minutes. It previously timed out at 10 minutes, and I didn't time it myself, so I changed it to 60 minutes to allow it to pass.

In this test, currently, each test function initializes a tokio runtime, so this may be causing the increase in execution time.